File: Java/bee 1043.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double a,b,c;
        a = sc.nextDouble();
        b = sc.nextDouble();
        c = sc.nextDouble();

        if(a < b + c && b < c + a && c < b + a){
            System.out.printf("Perimetro = %.1f\n", (a+b+c));
        } else{
            System.out.printf("Area = %.1f\n", ((a+b)*c)/2);
        }
    }
}

File: Ruby/bee 1043.rb
a,b,c = gets.split(" ").map { |n| n.to_f }

if a + b > c && a + c > b && b + c > a
  puts sprintf("Perimetro = %.1f", a+b+c)
else
  puts sprintf("Area = %.1f", ((a+b) * c)/2)
end